src/demo/DemoTestLoanService.java

```diff
@@ -2605,4 +2630,83 @@ public static void testDeleteSettlementAccount(SavingsAccount savingsAccount) th
 
 		System.out.println("The delete result is: " + deleteResult);
 	}
+
+
+	private static void testSearchLoanTransactionsWithCustomFields(LoanTransaction loanTransaction) throws MambuApiException {
+
+		String methodName = new Object() { }.getClass().getEnclosingMethod().getName();
+		System.out.println("\nIn " + methodName);
+
+		if(loanTransaction == null){
+			System.out.println("WARN: " + methodName + " could not run transaction is null");
```

Comment:
I would add "because" between "run" and "transaction" -> could not run because transaction is null

Reply:
good suggestion, thanks!